projects/plugins/jetpack/extensions/blocks/map/location-search/mapkit.js

```diff
@@ -30,6 +30,7 @@ const MapkitLocationSearch = ( { label, onAddPoint } ) => {
 					const withPlaceName = filtered.map( result => ( {
 						...result,
 						placeName: result.displayLines?.join( ', ' ),
+						original: result, // save the original result for later - otherwise mapkit.js gives a type error
```

Comment:
Interesting solution, but it works! 😁

Reply:
Believe me ... I tried more elegant ways 😁